src/app/static/src/tests/adrUpload/actions.test.ts

```diff
@@ -230,48 +173,37 @@ describe("ADR upload actions", () => {
                 resourceId: "123",
                 lastModified: "2021-03-02",
                 resourceUrl: "http://test"
+            },
+            shape: {
+                index: 2,
+                displayName: "shape",
+                resourceType: "adr-shape",
+                resourceFilename: "test-shape.geojson",
+                resourceName: "Naomi Input Shape file",
+                resourceId: "456",
+                lastModified: "2021-04-02",
+                resourceUrl: "http://adr/test-shape"
+            },
+            programme: {
+                index: 3,
+                displayName: "ART",
+                resourceType: "adr-art",
+                resourceFilename: "test-program.csv",
+                resourceName: "Naomi Input ART file",
+                resourceId: "789",
+                lastModified: "2021-04-04",
+                resourceUrl: "http://adr/test-art"
             }
         };
 
-        if (enableFeatureSwitch) {
-            expectedUploadFiles = {
-                ...expectedUploadFiles,
-                shape: {
-                    index: 2,
-                    displayName: "shape",
-                    resourceType: "adr-shape",
-                    resourceFilename: "test-shape.geojson",
-                    resourceName: "Naomi Input Shape file",
-                    resourceId: "456",
-                    lastModified: "2021-04-02",
-                    resourceUrl: "http://adr/test-shape"
-                },
-                programme: {
-                    index: 3,
-                    displayName: "ART",
-                    resourceType: "adr-art",
-                    resourceFilename: "test-program.csv",
-                    resourceName: "Naomi Input ART file",
-                    resourceId: "789",
-                    lastModified: "2021-04-04",
-                    resourceUrl: "http://adr/test-art"
-                }
-            };
-        }
-
         expect(commit.mock.calls[1][0].payload).toStrictEqual(expectedUploadFiles);
     };
 
-    it("getUploadFiles includes non-ADR input files when adrPushInputs feature switch is on", async () => {
-        await testGetUploadFilesInputs(true);
-    });
-
-    it("getUploadFiles does not include non-ADR input files when adrPushInputs feature switch is off", async () => {
-        await testGetUploadFilesInputs(false);
+    it("getUploadFiles includes non-ADR input files", async () => {
```

Comment:
Misleading test name now - all input files should be included, not just non-ADR ones
```suggestion
    it("getUploadFiles includes input files", async () => {
```